packages/palette/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@leafygreen-ui/palette",
+  "version": "0.9.0",
+  "description": "LeafyGreen UI Palettes",
+	"main": "./dist/index.js",
+	"types": "./dist/index.d.ts",
+  "scripts": {
+		"build": "../../node_modules/.bin/webpack --config ../../webpack.config.js",
```

Comment:
This isn't formatted according to prettier.

Let's update `prettier` and `prettier:check` to include `json` as an extension it checks to prevent future regressions.

Reply:
Fixed and ran prettier!

Reply:
This looks still wrong to me in the diff....But it passes the checks...but if I check it out locally and run format in my editor it does change it.

Reply:
👍